### ISSUE TITLE
Solving endogenous labor with population dynamics

### ISSUE THREAD

**Issue by adibalaram** [jdebacker/ProblemSet9#14]
Since Chapter 8 accounts for population dynamics but not endogenous labor, I presume we're going to have to solve the household's optimization problem and obtain the FOCs on our own.  u(c) would actually be u(c, n) and we'd end up with FOCs that aren't identical to what is already provided in Chapters 4 and 8. @jdebacker Am I right?

**Comment by jdebacker** [comment]
That's right -- in that you are combining chapters 4 and 8.  But it turns out that the FOC for the choice of labor will be of the same as that presented in Chap 4 (this is because both sides of the FOC refer to variables at time `t` and thus as divided through by the same things when the model is stationarized).
